pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/MercuryOrderStatusListener.java

```diff
@@ -11,26 +11,33 @@
 import com.google.gson.Gson;
 import com.google.pubsub.v1.ProjectSubscriptionName;
 import com.google.pubsub.v1.PubsubMessage;
+import lombok.extern.slf4j.Slf4j;
 import org.broadinstitute.dsm.db.dao.mercury.MercuryOrderDao;
+import org.broadinstitute.dsm.exception.DSMPubSubException;
 import org.broadinstitute.dsm.model.mercury.BaseMercuryStatusMessage;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
+@Slf4j
 public class MercuryOrderStatusListener {
-    private static final Logger logger = LoggerFactory.getLogger(MercuryOrderStatusListener.class);
 
+    private MercuryOrderStatusListener(){
+        throw new IllegalStateException("Utility class");
+    }
     public static void subscribeToOrderStatus(String projectId, String subscriptionId) {
         // Instantiate an asynchronous message receiver.
         MessageReceiver receiver = (PubsubMessage message, AckReplyConsumer consumer) -> {
+            String messageId = message.getMessageId();
             // Handle incoming message, then ack the received message.
-            logger.info("Got STATUS message with Id: " + message.getMessageId());
-
+            log.info(String.format("Got STATUS message with Id: %s", messageId));
```

Comment:
You should use the `{}` string replacement provided by the logger.
Some context in the message would be helpful. Which status does this refer to?

Reply:
But sonar tells me to not use `{}` :lolsob